exchanges/withdraw/withdraw.go

```diff
@@ -0,0 +1,71 @@
+package withdraw
+
+import (
+	"errors"
+	"strings"
+
+	"github.com/thrasher-corp/gocryptotrader/common"
+)
+
+// Valid takes interface and passes to asset type to check the request meets requirements to submit
+func Valid(request interface{}) error {
+	switch request := request.(type) {
+	case *FiatRequest:
+		return ValidateFiat(request)
+	case *CryptoRequest:
+		return ValidateCrypto(request)
+	default:
+		return errors.New("invalid request type")
+	}
+}
+
+// ValidateFiat checks if Fiat request is valid
+func ValidateFiat(request *FiatRequest) (err error) {
```

Comment:
Please add tests for these new funcs (super awesome BTW!)

Reply:
Coverage has been added for package :tada:

Reply:
💯